app/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
           android:versionCode="106"
-          android:versionName="2.53">
+          android:versionName="2.53.1">
```

Comment:
I think the versionName tags shouldn't change on backmerges, right?

Reply:
Correct, let me fix this